docs/source/features/sheet_management/index.rst

```diff
@@ -4,3 +4,126 @@
 ##################
  Sheet Management
 ##################
+
+In the Graviti SDK, ``Sheet`` and ``DataFrame`` are interpretations of the same thing at
+different levels.
+
+Sheet refers to the form of data organization that is one level lower than the dataset. One
+dataset can have many different sheets, such as train, test, or frame-by-frame pictures from
+different videos. Each sheet has its own schema.
+
+Graviti SDK organizes the data of a sheet into a DataFrame format, which makes it more convenient
+and intuitive to get and modify the data.
+
+More details about the ``Sheet`` and ``Schema`` are as follows:
+
+.. toctree::
+   :maxdepth: 1
+
+   dataframe
+   schema
+
+The following will introduce the sheet-related operations in the SDK. Of course, it is
+necessary to get a dataset first:
+
+.. code:: python
+
+   from graviti import Workspace
+
+   ws = Workspace(f"{YOUR_ACCESSKEY}")
+   dataset = ws.datasets.get(f"{DATASET_NAME}")
+
+****************
+ Create a Sheet
+****************
+
+Create a sheet on the open draft:
+
+.. code:: python
+
+   from graviti import DataFrame
+
+   draft = dataset.drafts.create(f"{DRAFT_TITLE}")
+   draft[f"{SHEET_NAME}"] = DataFrame(data=data, schema=schema)
+   draft[f"{SHEET_NAME}"] = DataFrame(schema=schema)
+   draft.upload()
+
+.. note::
+   Since only open drafts can upload the changes to sheet, it is better to use draft instead
+   of dataset for sheet modification operations.
+
+.. important::
+   If the sheet with the corresponding name already exists, SDK will delete the original one
+   and create a new one.
+
+*************
+ List Sheets
+*************
+
+List sheets of a certain version of the dataset:
+
+.. code:: python
+
+   list(dataset.keys())
+
+List sheets of a open draft:
+
+.. code:: python
+
+   draft = dataset.drafts.get({DRAFT_NUMBER})
+   list(draft.keys())
+
+SDK also supports listing sheets of the commit/tag/branch.
+
+.. code:: python
+
+   commit = dataset.commits.get()
+   list(commit.keys())
+
+   branch = dataset.branches.get(f"{BRANCH_NAME}")
+   list(branch.keys())
+
+   tag = dataset.tags.get(f"{TAG_NAME}")
+   list(tag.keys())
+
+*************
+ Get a Sheet
+*************
+
+Get a sheet of a certain version of the dataset:
+
+.. code:: python
+
+   dataset.checkout(f"{COMMIT_ID}")
+   dataset[f"{SHEET_NAME}"]
+
+Get a sheet of a open draft:
+
+.. code:: python
+
+   draft = dataset.drafts.get({DRAFT_NUMBER})
+   draft[f"{SHEET_NAME}"]
+
+SDK also supports getting the sheet of the commit/tag/branch.
```

Comment:
Is this duplicated with line 93?

Reply:
The description needs some changes. But it's not repetitive. What I want to express here is that user can checkout and get dataset sheet or get commit and get commit sheet.